base/interactiveutil.jl

```diff
@@ -701,15 +701,19 @@ end
 # testing
 
 """
-    runtests([tests=["all"] [, numcores=ceil(Int, Sys.CPU_CORES / 2) ]])
+    Base.runtests(tests=["all"], numcores=ceil(Int, Sys.CPU_CORES / 2);
+                  resilient=false)
 
 Run the Julia unit tests listed in `tests`, which can be either a string or an array of
-strings, using `numcores` processors. (not exported)
+strings, using `numcores` processors.
+If one test fails, all remaining tests will be discarded, unless resilient is `true`.
```

Comment:
Maybe state here explicitly that if `resilient = true` all remaining tests *in other files* will be run?

Reply:
Right, will clarify.